sys/include/ztimer.h

```diff
@@ -301,13 +301,26 @@ struct ztimer_base {
     uint32_t offset;            /**< offset from last timer in list */
 };
 
+/**
+ * @defgroup    ztimer_now64 ztimer_now64
+ * @deprecated  use uint32_t or ztimer64, will be removed after 2022.07 release.
+ *
+ * @brief deprecated module changes deprecated ztimer_now_t type
+ *
+ * use ztimer_now() returning uint32_t or alternatively use module ztimer64
+ * ztimer64_now() returning uint64_t.
+ * @{
```

Comment:
This makes `ztimer_now_t` part of the now deprecated `ztimer_now64` module (but Doxygen will render the `uint32_t` based version as `ztimer_now_t`). Maybe just leave this group empty:

```suggestion
 * @{
 * @}
 */
```

...

Reply:
i can just remove that `@{ @}`entirely (i thought that would help); but i also did not think that i would add more  documentation to a feature while deprecating its use.